packages/editor-ui/src/router.ts

```diff
@@ -476,10 +476,7 @@ export const routes = [
 					settingsView: SettingsUsersView,
 				},
 				meta: {
-					middleware: ['authenticated', 'role'],
-					middlewareOptions: {
-						role: [ROLE.Owner],
-					},
+					middleware: ['authenticated'],
```

Comment:
Do we want to allow regular members to access this page? If not, we should set `role: [ROLE.Owner, ROLE.Admin]` instead.

Reply:
I'm still not sure, the backend returns `user:list` and `user:read` scopes for both `member` and `admin` and it meant to me that everybody can see the page

Reply:
Hmm, ok, that's fair. I think it's required for sharing. Thanks for clearing it up. 

Just please ask JC whether accessing this page is an intended side effect.